src/App.js

```diff
@@ -0,0 +1,46 @@
+import './css/App.css';
+import { useEffect, useState } from 'react';
+import { HeaderArea } from './HeaderArea';
+import { TopBodyArea } from './TopBodyArea';
+import { ContentsArea } from './ContentsArea';
+import { FooterArea } from './FooterArea';
+import { getLoginUserInfo, getFolder } from './apis';
+
+function App() {
+  const [userInfo, setUserInfo] = useState([]);
+  const [links, setlinks] = useState([]);
+  const [folder, setFolder] = useState([]);
+
+  const handleLoad = async () => {
+    const userInfo = await getLoginUserInfo();
+    setUserInfo(userInfo);
+
+    const { folder } = await getFolder();
+    setFolder(folder);
+    setlinks(folder.links);
+  };
+
+  useEffect(() => {
+    handleLoad();
+  }, []);
+
+  return (
+    <div>
+      <header>
+        <HeaderArea email={userInfo.email}></HeaderArea>
+      </header>
+      <body>
+        <TopBodyArea
+          name={userInfo.name}
+          folderName={folder.name}
+        ></TopBodyArea>
+        <ContentsArea links={links}></ContentsArea>
+      </body>
+      <footer>
+        <FooterArea></FooterArea>
+      </footer>
+    </div>
```

Comment:
- 레이아웃 상 의도된 부분이 아니라면, 특별한 스타일이 없는 태그로 감싼 경우 Fragment(빈 태그)로 대체하시면 됩니다.
Fragment는 DOM에 실제로 그려지지 않기 때문에 렌더링 성능에 영향을 줄 수 있습니다!
- 자식 요소(children)를 받는 태그가 아니라면, self-closing해주세요!
- 이미 index.html에서 id가 root인 body 태그로 감싸고 있기 때문에, body 태그는 사용하시면 안됩니다.
- header 태그와 footer 태그는 각 컴포넌트 최상단에서 감싸주는 태그에 사용하는 게 좋겠네요!
```suggestion
    <>
      <HeaderArea email={userInfo.email} />
      <TopBodyArea
        name={userInfo.name}
        folderName={folder.name}
      />
      <ContentsArea links={links} />
      <FooterArea />
    </>
```